example/basic/main.go

```diff
@@ -69,6 +69,22 @@ func main() {
 
 		gauge.Set(ctx, 1)
 
+		meter.RecordBatch(
+			// Note: call-site variables added as context tags:
+			tag.NewContext(ctx,
+				tag.Insert(anotherKey.String("xyz"))),
+
+			metric.Measurement{
```

Comment:
Hm, is there no possibility to define values for the predefined keys when doing batched measurements?

Reply:
The handles contain the pre-defined label values. There's no ability to set additional labels on the call site, as agreed to in the working session.